src/js/App/Header/HeaderTests/Brand.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import ConnectedBrand, { Brand }  from '../Brand';
+import toJson from 'enzyme-to-json';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import { shallow } from 'enzyme';
+
+describe('Brand', () => {
+    let initialState;
+    let mockStore;
+    beforeEach(() => {
+        mockStore = configureStore();
+        initialState = {
+            chrome: {
+                navHidden: true
+            }
+        };
+    });
+
+    it('should render correctly with initial state', () => {
+        const store = mockStore(initialState);
+        const wrapper = shallow(
```

Comment:
Can you use mount for connected components? It will generate a lot more snapshots so we know everything is rendering correctly.

Reply:
![image](https://user-images.githubusercontent.com/28284746/69653599-7af4be80-1041-11ea-9cf3-a8a4706f40f9.png)

Reply:
![image](https://user-images.githubusercontent.com/28284746/69653689-995aba00-1041-11ea-9304-57b38ff758b6.png)

Reply:
so i tried to do this at first but I got this error that I couldn't fix and the enzyme docs/github issues seem to suggest that shallow with store/provider included has the same functionality.

Reply:
done